spacy_transformers/data_classes.py

```diff
@@ -261,9 +260,8 @@ def empty(cls, nr_docs) -> "FullTransformerBatch":
         return cls(
             spans=spans,
             wordpieces=WordpieceBatch.empty(),
-            tensors=[],
+            tensors=ModelOutput(last_hidden_state=torch.FloatTensor((0,))),
```

Comment:
Is it ideal that this is a tensor? Isn't the last hidden state really nothing rather than 0? I could imagine a pipeline using the last hidden state in a model and then accidentally tossing this tensor could lead to unintended consequences. I would use None instead.

Reply:
Good point, thanks!